Update stanov: dálkové hlasování, vedlejší činnost, kosmetické úpravy

jficz opened the pull request:
poznámka právníka k bodu 4. §11a (schvalování programu schůze na dálku):

> Zvažte tento přístup. Prakticky se nehlasuje o návrhu, ale nečinností dojde ke schválení. Doporučoval bych stanovit usnášeníschopnost (například opět pětinu) a k přijetí usnesení prostou většinu.

Záleží, o co nám jde víc - jestli o rychlost zahájení schůze nebo o to, aby ten program lidi opravdu s nějakým mandátem schválili.  V prvním případě se to dá řešit naplánováním (schůze prostě bude začínat "dřív"), v druhym případě riskujeme, že se ten program neschválí vůbec - což ale v souvislosti se změnou usnášeníschopnosti (update bodu 2. §11) asi zas tak velkou roli nehraje.

Současný návrh znění bodu 4. §11a (jako ostatně většina textu §11a a b) je moje tvorba, ale nijak zvlášť na tomhle znění netrvám.

Commit:
Update stanov: dálkové hlasování, vedlejší činnost, kosmetické úpravy

Roman3349 committed:
Oprava překlepů v nových stanovách

Signed-off-by: Roman Ondráček <ondracek.roman@centrum.cz>

repli2dev commented on stanovy_vpsFree.cz.md:
```diff
@@ -426,9 +481,12 @@ a dalších údajů, které mají být používány ke komunikaci Spolku s člen
 zveřejněno aktuální znění stanov.
+3. Tyto stanovy byly schváleny na jednání Schůze členů dne 8.3.2019 a
+nahrarazují v plném rozsahu stanovy Spolku ze dne 21.12.2015.
+4. Tyto stanovy jsou účinné od 8.3.2019.
```
Nemělo by zde být spíše 9.3.2019 (tzn. sobota?)

jficz replied:
mělo :)

repli2dev commented on stanovy_vpsFree.cz.md:
```diff
@@ -176,6 +188,48 @@ nápravu nepřesností ještě před rozesláním.
+1. Jednání Schůze členů dle § 11 může probíhat způsobem umožňujícím dálkovou
+účast; taková schůze může probíhat částečně či zcela ve virtuálním prostoru
+prostřednictvím prostředků komunikace na dálku, a to v informační systému
+(prostředí) Spolku, event. jiným způsobem (naříklad emailem).
```
Doplnil bych `jiným, avšak dostatečně důvěryhodným způsobem`, e-mail za takový nepovažuji, a proto bych jej odstranil z příkladu.

jficz replied:
Toť otázka. Když tam napíšeš "dostatečně důvěryhodným způsobem", tak si moc nepomůžeš, protože stejně pak někde jinde budeš muset tu "dostatečnou důvěryhodnost" definovat.

Mailem, když se použije nějakej rozumnej algoritmus (Oskar by určitě věděl jakej :) ), to může být klidně taky "dostatečně důvěryhodné". 

Spíš bych tam místo toho příkladu napsal něco jako "jiným způsobem stanoveným  radou nebo vnitřním předpisem".

davidkarban replied:
Co emaily podepsané gpg klíčem, který by bylo možné nahrát do vpsfree rozhraní u člena?

jficz replied:
tohle jsou imho technikálie, který patří když už někam, tak do toho interního předpisu (viz nový bod 1-4. a 6., §18)

Dacesilian replied:
Myslím si, že není možné nutit členy pořizovat si gpg klíč, vůbec se s ním učit a něco složitě nastavovat. Souhlasím každopádně s tím, že e-mail není věrohodný způsob a vpsAdmin po přihlášení by to řešil elegantně.

jficz replied:
Commit 54a270b, konkrétní technický řešení do stanov (imho) nepatří.

Jestli máte někdo jinej názor (a návrh konkrétního znění), sem s ním ;) Jinak to zavřu.

repli2dev replied:
Souhlasím, je to technikálie, která patří jinam. Takže zvolené řešení je fajn.

repli2dev reviewed (commented):
Nice :-)

cptMikky committed:
Merge pull request #1 from Roman3349/update-stanov

Oprava překlepů v nových stanovách

Commit:
fix datum schůze

Commit:
lepší wording pro hlasování mimo *řádnou* schůzi (§11b bod 6)

graywolf commented on stanovy_vpsFree.cz.md:
```diff
@@ -43,6 +43,18 @@ standardů.
+ve znění pozdějších předpisů, v rámci živnosti volné „Výroba, obchod a služby
+neuvedené v přílohách 1 až 3 živnostenského zákona.
+3. Veškerý zisk získaný Spolkem z vedlejší činnosti musí být použit v plném
+rozsahu na hlavní činnost Spolku.
```
ianal ale znamena tohle, ze zisk z vedlejsi cinnosti nelze pouzit na jinou vedlejsi cinnost? Je to umyslne omezeni?

jficz replied:
Ano, to je schválně. Cílem je zamezit jakékoliv komercializaci spolku.

Dacesilian commented on stanovy_vpsFree.cz.md:
```diff
@@ -176,6 +188,48 @@ nápravu nepřesností ještě před rozesláním.
+1. Jednání Schůze členů dle § 11 může probíhat způsobem umožňujícím dálkovou
+účast; taková schůze může probíhat částečně či zcela ve virtuálním prostoru
+prostřednictvím prostředků komunikace na dálku, a to v informační systému
+(prostředí) Spolku, event. jiným způsobem (například emailem).
```
```suggestion
(prostředí) Spolku, event. jiným způsobem (například e-mailem).
```

jficz replied:
vyhodil jsem to úplně (commit 54a270b)

Commit:
překlepy, kosmetika, wording, konzistence; *žádné změny* významu nebo smyslu jednotlivých ustanovení

Commit:
upravena definice způsobu hlasování na dálku (vypuštěn e-mail, nastavena definice Radou nebo řádem)

Commit:
přidána možnost schválit jednací a volební řád v rámci schvalování programu schůze

schvalování řádů je ve stanovách zakotvené, ale pro dálkové hlasování by to byly zbytečné blockery

Commit:
přidána zmínka o příjech z vedlejší činnosti do § o hospodaření

Commit:
odstraněna redundantní věta (imho nedává smysl)

jficz commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
+4. Program Schůze členů, která probíhá částečně či zcela na dálku, se považuje
+za schválený, pokud nedojde k jeho zamítnutí účastníky schůze do 1 hodiny od
+zahájení schůze; bod 5. § 11b o dálkovém hlasování se v tomto případě
+neuplatní.
```
K tomuhle bych ocenil nějaké alternativní návrhy.

repli2dev commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
   3. přestal splňovat požadavky na funkci člena Rady nebo Kontrolní komise a
-přesto neodstoupit z funkce.
-Mimořádná schůze členů musí projednat zejména toto podezření.
+  přesto neodstoupit z funkce.
```
```suggestion
  přesto neodstoupil z funkce.
```

repli2dev commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
+účast; taková schůze může probíhat částečně či zcela ve virtuálním prostoru
+prostřednictvím prostředků komunikace na dálku, a to v informační systému
+(prostředí) Spolku, event. jiným způsobem stanoveným Radou spolku nebo
+příslušným internim řádem
```
```suggestion
příslušným interním řádem
```

Commit:
fix typo

Co-Authored-By: cptMikky <cptMikky@users.noreply.github.com>

Commit:
fix typo

Co-Authored-By: cptMikky <cptMikky@users.noreply.github.com>

snajpa commented:
David Cesal mi psal mailem, je to dost relevantni k diskuzi, hodim to sem, snad se nebude zlobit:
> > On 2019-⁠03-⁠08 20:07, David Česal wrote:
> > Ahoj,
> > 
> > když teď řešíme změnu stanov a směřování spolku do budoucna,
> > položili jste si někdy otázku -⁠ proč jsme vlastně spolek?
> > 
> > Já jsem to na pražské schůzi nechtěl rozebírat nahlas, protože
> > tam byl zmatek ohledně formulací a neformální diskuse po schůzi
> > jsem se už neměl čas zúčastnit.
> > 
> > _Proč jsme spolek, proč ne firma? Proč to ti lidé dělají
> > (zadarmo/⁠bez zisku)? Není to už jen nějaká setrvačnost/⁠nebo je to
> > naopak právě ta forma, co pomáhá přitáhnout nové „členy"?_
> > 
> > _ _
> > 
> > Pokládám si tyhle otázky od chvíle, kdy jsem si pořizoval
> > připojení k internetu právě od místního spolku. Sice platíme
> > členské příspěvky a spolek se kryje pořádáním „školení",
> > ale členové jsou obyčejní zákazníci konzumující službu a nic
> > víc od spolku nečekají.
> > 
> > vpsFree to má podobně -⁠ ceník máme skrytý za „výši
> > členských příspěvků v závislosti na navýšení prostředků
> > člena", drtivá většina lidí na schůze nechodí a jediné, co
> > vidí, je funkční VPS. Místo školení tu máme podporu open
> > source. Když si přečtu, že „_Organizace vznikla jako přímá
> > reakce na nepružnost a přemrštěné ceny komerčních hostingů v
> > ČR_", tak to určitě mohlo platit dříve, ale dnes už to neplatí.
> > 
> > Vývoj do budoucna
> > 
> > Protože se situace změnila a vás už asi také nebaví jen
> > přidávat další servery, rozhodli jste se, jak jsem to pochopil,
> > vytvořit druhý Microsoft Azure -⁠ účtování po minutách, výkon
> > na vyžádání. To už budete účtovat firmám jako komerční
> > službu a nebude asi žádný důvod, aby cena byla nějak výrazně
> > nižší než u těch původně kritizovaných komerčních hostingů.
> > Jenže v tu chvíli nemáme vůbec schopnost konkurovat -⁠ dvě ISO
> > certifikace spočívající v sepsání pár papírů nenahradí
> > třeba stovky bezpečáků sedících v datacentru Microsoftu; o SLA
> > ani nemluvě.
> > 
> > Dělat něco zadarmo
> > 
> > Sám dělám předsedu SVJ v našem paneláku. Odměnu máme asi 5
> > tisíc za rok, ale je jasné, že vzhledem k mým jiným možnostem a
> > s přihlédnutím k rizikům plynoucím z funkce, bych to nechtěl
> > dělat ani za 15 tisíc měsíčně. Proč to dělám? Jedno
> > vysvětlení přeci jen může být, že si dávám pozor na svou
> > část majetku. Druhým je ale fakt, že domu plnému starších lidí
> > pak už zbývá si jen najmout někoho externího a to není to
> > pravé.
> > 
> > Mám málo času na práci, za kterou mi chce někdo platit, tak
> > nemám vysvětlení pro to dělat něco zadarmo. Jedna věc je
> > prospěšná činnost (jít sbírat odpadky do lesa, člověk se o
> > víkendu alespoň protáhne) a jiná věc budovat spolek založený na
> > naprosto komoditní věci (virtuální servery) a vedle firem
> > nabízejících to samé za stejnou cenu.
> > 
> > Podle stanov vám odměnu stanovuje kontrolní komise. Kdybych vaše
> > odměny znal (nechci), třeba by to vysvětlilo tu motivaci : )
> > Protože jestli to děláte „zadarmo" (a to asi děláte, protože
> > milion ročně odměny jednoho člověka je „tak akorát"), moc vás
> > nechápu a právě na pochopení tenhle e-⁠mail cílí.
> > 
> > „Můj spolek"
> > 
> > Leda byste si to vybírali jinak (kromě odměn) a celkově by se vám
> > to vyplatilo. Mě by se například hodil load balancing/⁠fail over a z
> > pohledu člověka, co má k dispozici celou infrastrukturu by to nebyl
> > žádný problém zařídit. Určitě chápete mojí myšlenku. Jenže
> > to není čisté, je to zneužití pozice. Ale je to jedno, protože
> > členové to nezjistí a případně omluví.
> > 
> > Také bych měl vůbec problém budovat něco, co mi může někdo
> > vzít (schůze odvolat) /⁠ není to moje. To máte elegantně řešeno
> > ve stanovách mandátem na doživotí, ale alespoň teoreticky tam
> > pořád existuje možnost odvolání člena Rady.
> > 
> > Stojí vám to za to?
> > 
> > Asi si pořád dokážete najít vysvětlení, proč to děláte. Já
> > bych to nedokázal, proto vám to píšu a třeba budete mít chuť mi
> > na to něco napsat : ) Případně se můžeme pobavit, když bude
> > někdy schůze zase v Praze.
> > 
> > Mám servery, které mohu do nekonečna spravovat; mohu si hrát skoro
> > se vším, co mě napadne. Ale zpětně vidím, kolik času jsem
> > věnoval něčemu neproduktivnímu/⁠projektu do šuplíku a co jsem
> > mohl dělat místo toho. Proto když mám věnovat čas sezení u
> > počítače, musím už dělat jen to, za co mi někdo zaplatí.
> > Budovat spolek asi není zábava na hodinu týdně.
> > 
> > Výhody spolku
> > 
> > Jak jsem pochopil, výhody spolku jsou v tom, že si lidé řeknou -⁠
> > _spolek, tam to dělají nadšenci ve svém volném čase, podpořím
> > je_._ _Vzhledem k výše napsané úvaze to asi zadarmo nebude, ale to
> > už lidé nevidí; alespoň mají dobrý pocit, asi je to výhra pro
> > obě strany.
> > 
> > Četl jsem o možnosti reklamy zadarmo pro neziskové organizace, to
> > by možná stálo za to zkusit -⁠
> > https://support.google.com/nonprofits/answer/3215869?hl=cs&ref_topic=3247288
> > [1] .
> > 
> > Shrnutí mailu je -⁠ nemám nic proti vám, děláte to dobře, jsem za
> > to rád. Mám jen otázku -⁠ proč si prostě nezaložíte firmu?
> > Vyhnuli byste se tím problémům se svoláváním schůzí, nemuseli
> > byste nikomu nic vysvětlovat a dělali byste to samé.
> > 
> > Pěkný víkend
> > 
> > David
> > 
> > Links:
> > -⁠-⁠-⁠-⁠-⁠-⁠
> > [1]
> > https://support.google.com/nonprofits/answer/3215869?hl=cs&amp;ref_topic=3247288
> Zdarec Davide,
> 
> ono pro to, co je vpsFree, asi zadna idealni pravni forma neexistuje, vzdycky to bude ohybani cehosi, co na to, co to ma byt, sedi aspon vzdalene.
> 
> To, co delame technicky, je jenom malinko, malilinkato z toho, co bychom delat chteli... je to na trochu delsi filozofickou debatu o smyslu zivota a o tom, s cim kdo z nas chce umirat, co maji byt ty posledni veci, co stihneme rict tem naokolo...
> 
> vpsFree je pro mne osobne tak 10% z toho, co chci za zivot dosahnout, to, ze je to neziskovka, ve svy forme, tak jak to je, patri vam vsem clenum. Nevite o tom, ale pomahate mi stavet neco mnohem vetsiho, nez je samotny vpsFree. Ono se to sklada postupne, ale uz konecne zacina prihustovat a zacinaji bejt videt konkretni obrysy, jak bude vypadat to, co tu na ty planete necham.
> 
> Nechci nic z toho zakriknout, je za tim proste mnohem vic, nez par serveru a snazit se napodobit Azure :)
> 
> Primarne potrebuju okolo sebe uzivit co nejvic chytrejch lidi tak, aby nemuseli resit, kolik stoji rohlik, maslo, ale ani kolik stoji auto, nebo nejaka nadherna dovolena s nejakou krasnou pritelkyni - kdyz uz na to ti lidi, co se mnou jdou a kazdy den makaji, budou mit cas.
> 
> To, co tu po sobe nechame, kdyz mi to nikdo nezkurvi driv, nez to stihne ty obrysy dostat, bohate vykompenzuje docasna nepochopeni :D
> 
> Nedelame to pro sebe, chceme budovat rodiny, ktery makaji pro ostatni.
> 
> Snazim se uz 10 let ukazat ostatnim, ze je mozny se dost slusne uzivit sluzbou ostatnim, obzlvast pokud je za tim snaha dostat civilizaci na dalsi civilizacni stupen.
> 
> V silencich typu Elon Musk je podle mne budoucnost - Elon taky nejede na to, aby mu vsechno patrilo, podle mne si spis musi kazdej den mlatit do cela, ze jak je mozny, ze mu vic lidi nechce pomoct tim spravnym smerem - vsichni se k nemu hrnou jako zamestnanci.
> 
> A to ja nechci.
> 
> Ja chci zivotni spolupracovniky, se kteryma tu nechame kus uzitecny prace vsem - co nam mimojine na jisto uzivi rodiny. Ale valime ciste open-source, ciste na otevrenych principech, pod heslem nezakazovat druhym vedet, jak se co dela...
> 
> Kdyz se mame dostat na nejakou rozumnou civilizacni uroven, jak postavit elektrofuzni reaktor musi umet kazdy decko ;)
> 
> vpsFree je o komunikacich, o tom, abychom meli takto podchycene otevrene komunikace a nemohl nam nikdo vzit svobodnou moznost vystavit neco na net jen tak - ale na to ta organizace musi mit nejakou silu.
> 
> A jedine, kdyz bude nestranna, bude mit slovo - ale podle mne neni spatne, aby mel v takove organizaci hlavni slovo zakladatel, dokud tomu ma co rict.
> 
> Jak rikam, je to tak desetina, mozna ani ne, z celeho zivotniho planu.
> 
> A potrebuju mit vcelku jistotu, ze mi to nikdo nerozboura, proto jsou ty stanovy ve smyslu BDFL. Obzvlast, pokud jsem na tom zatim docela existencialne zavisly - ale i az nebudu, bude vpsF mit uz takove rozpocty, ze bude mit bliz jednak k naplnovani nejakych tech vetsich cilu - ale taky bliz k tomu, aby to nekdo chtel kvuli penezum posrat.
> 
> Dava ti to uz vic smysl? ;)
> 
> Co si o tom myslis ty? :)
> 
> /⁠snajpa
>

jficz commented:
@snajpa tohle imho patří spíš na mailing list a/nebo ideálně fakt na nějakou face to face diskusi (který bych se sám docela rád zůčastnil :) ), ale tady bych radši zůstal jen u té změny stanov.

that being said, dokázal bych z rukávu vysypat nejmíň tucet důvodů, proč by vpsFree jako firma nemohlo dlouhodobě fungovat s takovými ideály, s jakými funguje teď (jestli vůbec) a myslim si, že svět potřebuje podobný spolky s podobnými principy (a to mam obecně s NGO problém).

Taky předsedám bytovému družstvu (a místopředsedám v SVJ), na kolejích jsem byl v představenstvu kolejního klubu a jsem členem dalších několika skutečně "obecně prospěšných spolků". Ten největší z nich funguje podobně jako vpsfree - má hafo členů a spolek vyvíjí činnost, která je pro členy prospěšná, je tam nějaký zdravý jádro, který ten spolek drží pohromadě a spousta členů, kteří jen posílají příspěvky a jinak nic. A funguje to. Velmi dobře - ten spolek patří k jedněm z nejrespekovanějších subjektům ve svém oboru a má v tomto směru obrovský vliv. 

vpsFree je na cestě stejným směrem. A to se mi líbí a hodlám to podporovat dál, ať už placením členských příspěvků, účastí na schůzích a nebo příležitostným přiložením ruky k dílu.

snajpa commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
+provedeno na řádné Schůzi členů
+7. Termín pro odevzdání dálkového hlasu podle předchozího bodu stanoví Rada spolku
+nejméně na 336 hodin; kratší termín může Rada spolku stanovit pouze v
+odůvodněných případech, nejméně však na 24 hodin
```
Prijde mi, ze tyhle lhuty jsou prilis kratke, nedovedu si predstavit oduvodneny pripad, kdy bychom mohli chtit provest rozumne uvazenou zmenu stanov, pripadne jinou zasadni zmenu ve fungovani - a mit pritom jasno, jake to bude mit dopady, behem 24 hodin...

Navrhuju termin odevzdani hlasu na 10 pracovnich dni, oduvodnene pripady nejmene 5 pracovnich dni.

jficz replied:
k termínům v bodu 5: každej bod musí být součástí programu schůze, tj. lidi by měli minimálně 14 dní předem vědět, o čem budou hlasovat. Těch 5 dní (120 hodin) je tam pak spíš technická lhůta proto, aby "to každej pohodlně stihnul" - účel týhle lhůty neni dát lidem čas na rozmyšlenou, ale čas "se k tomu dostat". Ale souhlas s tím, že by se tam mělo doplnit nějaký minimum, aby si rada nemohla říct "máte na to 10 sekund..."

k termíům v bodu 7: 336 hodin (14 dní) je tam právě proto, aby lidi měli stejnej čas, jako při vypsání schůze, tj. 14 dní. Zkrácení na 24 hodin v odůvodněných případech by měla řešit akutní průserový situace. Na druhou stranu, nedovedu si takovou situaci představit, imho tam můžeme dát i těch 5 dní (120 hodin).

lisaak1 replied:
Souhlas - 24 hodin na to o čem by se mohlo hlasovat je málo. Jen bych to asi nekomplikoval pracovními dny stačí přidat příslušný počet dní kvůli víkendům. 

Počítání v hodinách je asi taky lepší, nebude se muset řešit opičárna typu "X dnů následujících po dni započetí hlasování" apod.

jficz replied:
prodlouženo na minimálně 120 hodin, zpřísněna podmínka pro využití minimálního času

jficz replied:
partially resolved by 6018687 (lhůty v bodu 7)

jficz replied:
jak byste vyřešili ten bod 5.? Nechat tam pevnou lhůtu? Nebo dát radě možnost tu lhůtu jenom prodloužit? Tam by se pak taky musel dát nějakej strop...

lisaak1 replied:
IMHO pokud bude bezesporné odhlasování programu, tak to může zůstat jak to je (případná šibeniční lhůta do programu neprojde), pokud zůstane krátká lhůta pro zamítnutí programu, tak by to chtělo dát jen možnost prodloužení. Nesmí nastat situace, že by mohly být obě lhůty šibeniční. (a zase a) něco špatného někdo protlačí, b) někdo bude moci napadnout hlasování).

jficz replied:
souhlas, vyřešil bych to lepším schválením programu - to stejně potřebuje ještě potunit :)

snajpa commented on stanovy_vpsFree.cz.md:
```diff
@@ -305,107 +368,104 @@ systému Spolku nejméně 10 pracovních dnů před začátkem platnosti.
-Podepisování za Spolku se uskutečňuje tak, že k názvu Spolku se připojí jméno
-a podpis předsedy. V případě, že podepisuje pověřený člen Rady, je povinen
-doložit pověření od Rady.
-2. Předseda je odpovědný za celkovou koordinaci, vedení a provozní činnost
```
Je dobry napad tohle odstranit? Takhle bylo jasne, kdo ma podpisove pravo...

jficz replied:
neni to smazaný, ale přesunutý do §15 bod 2

lisaak1 replied:
Čekal bych že to bude spíš součástí zápisu v rejstříku, tak jako je to u firem, ale nenašel jsem to tam. Kde jinde by to mohlo být uvedeno?

jficz replied:
podrobně: za spolek jedná (tj. "podepisuje") statutární orgán, v našem případě Rada - to je implicitní. Ty, jako předseda, fakticky nejednáš za spolek, ale za Radu, která, in turn, jedná za spolek. "Jednání navenek" znamená všechny úkony s vnějším světem "jménem rady". Podepisování je v tom implicitně zahrnutý.

jficz replied:
@lisaak1: V rejstříku je to špatně, tam se to musí opravit i v návaznosti na nový stanovy.

jficz replied:
@snajpa dává ti to vysvětlení smysl?

snajpa replied:
Dava, ale takhle explicitne se to hodi, napr. kdyz se jedna o nejaky pujcky/bankovni sluzby, tem jde o co nejvetsi jistotu a tohle neskodi :)

jficz replied:
pro tyhle případy je obvykle rozhodující to, co je v rejstříku a v rejstříku to bude napsaný zhruba ve smyslu "předseda jedná jménem rady samostatně". 

Stanovy nikoho takhle moc zajímat nebudou (vlastní zkušenost).

Ale secondary layer "jistoty" imho asi snad ničemu neuškodí, zkusim to tam nějak naformulovat zpátky. Nicméně si  furt myslim, že předseda nemůže z principu jednat jménem spolku, protože pak by to byl de-facto druhej statutár vedle Rady.

jficz replied:
resolved by 4924e22, @snajpa vyhovuje takhle?

snajpa replied:
idealka

snajpa reviewed (commented):
review of a review of a review... we need to go deeper

(preklik)

lisaak1 commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
+5. Funkční období člena Rady není omezeno časově; člen Rady může být odvolán
 pouze Schůzí členů na jejím pravidelném nebo mimořádném zasedání. Výkon funkce
-člena Rady skončí dnem zvolením nového člena.
+člena Rady skončí dnem zvolením nového člena
```
Vím že to je jen syntaktická úprava, ale mám připomínku k originálnímu znění:
členů rady může být 3-5, hypotetická situace - máme 5 členů, schůze 2 členy odvolá, nikoho místo nich nezvolí. Podle mě by měli oba skončit ihned, Rada bude stále v souladu se stanovami. Pravděpodobně je odvoláme, protože je tam nechceme z nějakého závažného důvodu, nebo protože o to z nějakého závažného důvodu daný člen rady požádá. Tudíž by odvolaný člen rady měl skončit ihned. Proč by měli zůstat v radě dokud nezvolíme někoho jiného? To by mělo smysl pouze pokud by počet členů rady klesů pod 3.

jficz replied:
jo, na tohle jsem taky koukal, ale zase se mi nechtělo z týhle změny dělat úplnej overkill (ikdyž už to tak stejně začíná vypadat)

lisaak1 replied:
OK. Jen mi to přišlo zbytečně složité.

lisaak1 commented on stanovy_vpsFree.cz.md:
```diff
@@ -305,107 +368,104 @@ systému Spolku nejméně 10 pracovních dnů před začátkem platnosti.
-komise volí kooptovaného člena z těchto kandidatur.
+komise volí kooptovaného člena z těchto kandidatur
 8. Členům Kontrolní komise může příslušet za výkon jejich funkce odměna. O výši
 a způsobu výplaty odměny rozhoduje Rada Spolku. Členové Kontrolní komise mají
```
návrh: KK by měl být orgán zodpovědný členům (resp. členské schůzi). O případné odměně měla rozhodovat schůze a nikoliv rada. Takto odměny radě schvaluje KK a odměny KK schvaluje rada. Zde vidím potenciální závislost.

jficz replied:
taky overkill. Tahle změna by měla být zaměřená jen na legal cleanup, dálkový hlasování  a vedlejší činnost (tj. věci, který se probíraly na schůzi).

tohle bych klidně nechal na diskusi někdy do budoucna. Jakmile budeme moct hlasovat dálkově, nebude už problém ty stanovy doplnit kdykoliv v budoucnu.

lisaak1 replied:
OK, tohle je fakt na diskusi a jen by to zdržovalo.

lisaak1 commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
+hlasovat na dálku; takové hlasování má stejné účinky, jako hlasování v
+přítomnosti
+4. Program Schůze členů, která probíhá částečně či zcela na dálku, se považuje
+za schválený, pokud nedojde k jeho zamítnutí účastníky schůze do 1 hodiny od
```
Jestli je to tak, že zamítnout program lze pouze v intervalu <začátek schůze; začátek schůze + 1h>, tak navrhuji raději delší termín. Předpokládám, že jde o to zamezit nějakému "nepřátelskému aktu" a pokud je interval jen jedna hodina, tak mi to přijde krátké. To raději "začít" schůzi dříve.

jficz replied:
jde o to, aby "zahájení schůze" netrvalo 5 dní jenom proto, aby se na konci zjistilo, že neni schválenej program... Aktuální návrh řešení (1 hodina) není ideální, ale nic lepšího zatím nepadlo.

lisaak1 replied:
S tím souhlasím, ale hodinu můžeš strávit i v zácpě, do který jsi neplánoval spadnout. Když to bude 6 hodin, tak už je pravděpodobnost že nestihneš podat zamítnutí podstatně nižší. U té hodiny mi to přijde opravdu hodně pravděpodobný. 
Pravda je, že jsem zatím nevymyslel případ kdy by to mohlo opravdu uškodit i v případě nějakého nepřátelského úmyslu, protože i pokud by někdo protlačil takto nečistě nějaký "špatný" program jednání, tak se prostě neodhlasuje, protože na to je času dost. A pak můžeme řešit co s tím třeba přes KK. 
Nemůže to být nějak zneužitelné naopak ve smyslu, že se něco odhlasuje a někdo tu hodinovou lhůtu bude moct napadnout a prohlašovat pořád dokola, že nemohl zamítnout program a tím že je hlasování neplatné? Prostě takový sociální DoS?

domankus replied:
Tu bude asi dôležité čas začiatku hlasovania o programe schôdze. Ak by sa dalo začať hlasovať pár hodín pred samotným začiatkom schôdze, tak by to časové okno narástlo a neboli by problémy s krátkosťou času. V podstate by sa mohlo začať hlasovať o programe od zaslania pozvánky.

jficz replied:
Sociální DoS by musel ten dotyčnej řešit soudně. 

Osobně jsem tam dal ten schvalovací default hlavně proto, že minimálně v začátku teď netušíme, jestli to dálkový hlasování vůbec bude fungovat a kdyby se náhodou stalo, že fungovat nebude, byl by spolek kompletně paralyzovanej, protože by fakticky nemohl zahájit schůzi členů.

Souhlas že hodina je málo. Ale když už tam bude 6 hodin, může tam úplně stejně být 24 hodin a schůze se holt de-facto bude svolávat 15 dní předem a ne 14 s tím, že den před zahájením se musí schválit program.

Další možnost by byla řešit schválení programu bodem 7, ale tam neumim říct, jaký by to mělo potenciální legální konsekvence.

lisaak1 replied:
A proč není možné napsat, že se považuje za schválený pokud do začátku schůze není zamítnutý? Tj. že by bylo přímo ve stanovách, že během těch 14 dnů od odeslání pozvánky do začátku schůze se může program zamítnout a pokud se do začátku nezamítne, tak je považován za schválený. 

BTW co to znamená přesně "zamítnout program"? To znamená, že když libovolný jeden člen napíše, že program zamítá, tak je zamítnutý?

jficz replied:
Protože schválení programu je součástí schůze, tj. nejdřív se musí ta schůze zahájit, pak se teprve může hlasovat.

Zamítnutí je defaultní stav hlasování ve všech případech, kromě případu, kdy dojde ke schválení (tj. prostá většina přítomných členů hlasuje PRO). V případě schválení programu jsem tu logiku z výše popsaných důvodů otočil, tj. aby došlo k zamítnutí programu, musí prostá většina přítomných hlasovat PROTI. 

Opět, vím, že to neni ideální a přijímám návrhy na lepší řešení ;)

jficz replied:
fixed by 9213ceb. Limit 24 hodin, implicitní schválení zachováno

Commit:
zvednut minimální termín pro mimořádné per rollam hlasování na 120 hodin + zpřísněna podmínka

jficz commented:
sorry, neumim očividně zacházet s github review :) snad v tom nedělam moc velkej bordel

Commit:
definice způsobu podepisování Předsedy, upřesnění povinností při pověření

domankus commented on stanovy_vpsFree.cz.md:
```diff
@@ -132,13 +144,13 @@ Každý člen Spolku je povinen:
+  předchozí řádné schůze členů,
   4. schvaluje zprávu Kontrolní komise o revizi činnosti a hospodaření Spolku
-za období od předchozí chůze členů,
+  za období od předchozí řádné chůze členů,
```
Tu chýba "s" v slove "schůze".
```suggestion
  za období od předchozí řádné schůze členů,
```

domankus commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
 Schůze
-2. Řádně svolaná Schůze členů je vždy usnášeníschopná. Schůze členů rozhoduje 
-prostou většinou hlasů přítomných členů.
+2. Řádně svolaná Schůze členů je usnášeníschopná při účasti alespoň třetiny
```
Zišla sa niekedy na schôdzi aspoň tretina členov? Lebo toto by znamenal, že možno žiadna schôdza nebude uznášaniaschopná. Alebo to vychádza z legislatívy?

Dacesilian commented on stanovy_vpsFree.cz.md:
```diff
@@ -154,149 +166,200 @@ informačním systému spolku
-2. Řádně svolaná Schůze členů je vždy usnášeníschopná. Schůze členů rozhoduje 
-prostou většinou hlasů přítomných členů.
+2. Řádně svolaná Schůze členů je usnášeníschopná při účasti alespoň třetiny
+členů. Schůze členů rozhoduje prostou většinou hlasů přítomných členů.
```
Je to v pořádku - http://poradna.crdm.cz/clenska-schuze/usnasenischopnost-clenske-schuze-740 .

Dacesilian commented:
@snajpa @cptMikky Rád se s vámi v Praze potkám, až bude někdy v létě čas, klidně se ozvěte :)

Ať se na to podívám z jakékoli strany, tak mi to pořád vychází jako firma. Zabezpečit zaměstnance a jejich rodiny (dát jim velkou mzdu), vybudovat něco velkého. Řídit to shora úzkou skupinou lidí. Nevýhody spolku (usnášeníschopnost) teď zmizí s novými stanovami. V těch stanovách mimochodem stojí, že odměnu rady určuje kontrolní komise a odměnu kontrolní komise určuje rada. Neměla by to řešit členská schůze jako nejvyšší orgán? Ve stanovách nemáme žádná omezení částek, se kterými může kdo disponovat. Tam cítím, že stačí být zadobře s pár lidmi a vzájemně budou všichni v pohodě :) Teď tu vidím, že ty odměny @lisaak1 taky rozporuje.

Proč by ale měl někdo přijít "k hotovému"(stát se členem spolku) a rozhodovat o společných věcech? Opravdu spolek vnímáš, jak píšeš, že všechno je všech? To bych já osobně jako zakladatel nedokázal. Není zřejmé, zda to něco velkého buduješ uvnitř spolku nebo opravdu jen ty sám za sebe. Zda ti to pomáháme budovat jen těmi odměnami nebo nevím jak :)

Uvádím trochu rýpavé otázky, které u spolku asi každého napadnou. Není to nic osobního, pořád je to o té formě. Firma může mít stáže pro studenty, aby se něco nového naučili. Může mít nadaci, ze které bude podporovat bohulibé činnosti. Když se chce člověk zapojit do činnosti spolku, těžko bude rovnocenný zakladatelům.

Commit:
zvýšení limitu zamítnutí programu na 24 hodin, upřesnění pojmů

istana reviewed (approved):
lgtm

domankus committed:
fix small typo

chůze => schůze

Co-Authored-By: cptMikky <cptMikky@users.noreply.github.com>
